src/System.Private.CoreLib/shared/System/ComponentModel/DefaultValueAttribute.cs

```diff
@@ -36,7 +37,7 @@ public DefaultValueAttribute(Type type, string value)
             // We would fail to load an otherwise normal class.
```

Comment:
I believe null is a valid value for the `string value` parameter.

Reply:
It might even be for the `type` parameter.

Reply:
We have a catch statement to catch any type of exceptions but internally ```convertFromInvariantString``` throws when string or type is null

Reply:
Yes, but that's an implementation detail.